sdk/python/kfp_tekton/compiler/main.py

```diff
@@ -49,7 +49,7 @@ def parse_arguments():
   return args
 
 
-def _compile_pipeline_function(pipeline_funcs, function_name, output_path, type_check):
+def _compile_pipeline_function(pipeline_funcs, function_name, output_path, type_check:
```

Comment:
typo that fails the tests

Reply:
thanks!

Reply:
can you also fix the python lint? you can check the errors with `make lint`